pkgs/by-name/fr/frr/package.nix

```diff
@@ -124,6 +133,10 @@ stdenv.mkDerivation (finalAttrs: {
     net-snmp
   ] ++ lib.optionals (lib.meta.availableOn stdenv.hostPlatform elfutils) [
     elfutils
+  ] ++ lib.optionals grpcSupport [
+    grpc
+    protobuf
+    which
```

Comment:
Is `which` really a runtime dependency? If it's only used at builtime, this should go into `nativeBuildInputs`.